angular-mentoring/src/app/header/header.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="header__logo">
+    <img src="./../../assets/logo/logo.png" srcset="./../../assets/logo/logo@2x.png 2x,
```

Comment:
Absolute paths can be used here (`assets/logo/logo.png` instead of `./../../assets/logo/logo.png`)